test/integration/render/collection-point/strokeColor/scenario.js

```diff
@@ -1,11 +1,11 @@
-/* global carto geojson */
+/* global carto sources */
```

Comment:
instead duplicating this line in every file we can add a .eslint.js file at top level